src/scripts/placeDE-overlay.user.js

```diff
@@ -10,19 +10,17 @@
 // @downloadURL  https://github.com/PlaceDE-Official/place-overlay/raw/main/src/scripts/placeDE-overlay.user.js
 // ==/UserScript==
 
-var overlayImage = null;
-if (window.top !== window.self) {
-    window.addEventListener('load', () => {
-        const canvasContainer = document.querySelector("garlic-bread-embed").shadowRoot.querySelector("div.layout").querySelector("garlic-bread-canvas").shadowRoot.querySelector("div.container");
+let overlayImage = null;
+if(window.top !== window.self) {
+    window.addEventListener("load", function() {
+        const canvasContainer = document.querySelector("garlic-bread-embed").shadowRoot.querySelector("garlic-bread-canvas").shadowRoot.querySelector(".container");
         overlayImage = document.createElement("img");
+        function updateImage() {
+            overlayImage.src = "https://place.army/overlay_target.png?" + Date.now()
+        }
         updateImage();
-        overlayImage.style = `position: absolute;left: 0;top: 0;image-rendering: pixelated;width: 1000px;height: 1000px;`;
+        setInterval(updateImage, 30000);
```

Comment:
i would add a variable for this for better overview

Reply:
for the style?

Reply:
For the User to better notice.

If you use a variable such as 'refreshTimer' a user knows what this number is about even if they don't use a ide.

Reply:
true, but then i would also make style its own var

Reply:
all made to variable